.github/pull_request_template.md

```diff
@@ -1,17 +1,9 @@
-# Why was this change made? 🤔
+# Why was this change made?
 
+<!-- Feel free to use whatever GitHub copy/pastes into the PR description above, or even just `Fixes #1234` if the PR is closely tied to an issue. -->
```

Comment:
I think we'll even get sick of this.  How about:

"fixes #123 is fine if the PR is closely tied to an issue;  a few words about WHY if not.  (WHAT goes in the commit messages)".  ???

Reply:
Changed.